index.html

```diff
@@ -22,7 +22,7 @@
 	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 	<link href="https://emoji-css.afeld.me/emoji.css" rel="stylesheet">
 
-	<title>Engineer-Petr 🌎🚜🇷🇺🇺🇦🇧🇾</title>
+	<title>Engineer-Petr: Relocation for IT specialists 🌎🚜🇷🇺🇺🇦🇧🇾</title>
```

Comment:
As far as I understand, the term in english for "IT специалист" is a tech worker. So it may be good to call it that way and save "IT специалист" for the description in Russian.

Reply:
In NA, `specialist` is used, probably, more often than `worker`. What do you think about `IT professionals` instead?

Reply:
It professionals sounds great to me!